packages/cardano-services-client/scripts/createVersionSource.js

```diff
@@ -1,10 +1,27 @@
 const fs = require('fs');
 const path = require('path');
 
-const openApiSpec = require('../../cardano-services/src/Http/openApi.json');
+const cardanoServicesSrc = path.join('..', '..', 'cardano-services', 'src');
+
+const apiVersion = Object.fromEntries(
+  Object.entries({
+    assetInfo: 'Asset',
+    chainHistory: 'ChainHistory',
```

Comment:
I'm wondering if we could walk through fs and find all `(.+)HttpService` alongside openApi.json in the same dir. Then we wouldn't need to maintain this file separately and rely on convention.

Reply:
Right we can't get rid of the file, the idea was to not have to update this file when we add/remove/rename providers. Anyway it's good as-is.